include/struct_defs/struct_02090800_sub1.h

```diff
@@ -6,6 +6,6 @@
 typedef struct {
     int unk_00;
     Strbuf* unk_04;
-} UnkStruct_02090800_sub1;
+} PokemonInfoDisplayStruct_sub1;
```

Comment:
```suggestion
} PokemonInfoDisplay_sub1;
```

Also, is there enough info on this struct to document the fields?